### ISSUE TITLE
add a way to view next/previous content in `AccessibleView`

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#184612]
this would be useful in notebook output cells for example so that the user doesn't have to press `f2`, `escape`, `ctrl+down` then `f2` again 

cc @amunger

**Comment by amunger** [comment]
what is `f2` doing in this case?

**Comment by meganrogge** [comment]
presenting the content in a `monaco-editor`

**Comment by meganrogge** [comment]
this is now possible as there's an `onKeyDown` callback for the content provider

https://github.com/microsoft/vscode/blob/69defeced64987e1e7c2d3b2ea1df0903ae738bd/src/vs/workbench/contrib/accessibility/browser/accessibleView.ts#L24-L31

**Comment by meganrogge** [comment]
we should use `alt + [ ]` for this per @jooyoungseo to align with other experiences
